exonum-java-binding-qa-service/src/main/java/com/exonum/binding/qaservice/QaServiceImpl.java

```diff
@@ -88,9 +94,8 @@ protected Schema createDataSchema(View view) {
 
   @Override
   public Optional<String> initialize(Fork fork) {
-    // Add a default counter to the blockchain.
-    String defaultCounterName = "default";
-    new CreateCounterTx(defaultCounterName)
+    // Add an initial counter to the blockchain.
+    new CreateCounterTx(INITIAL_COUNTER_NAME)
```

Comment:
What was the purpose of previously existed counter ? Can you create new separate counter for your purposes ? Just to avoid potential mess.